tox.ini

```diff
@@ -68,6 +68,10 @@ commands =
 	 # file.  See pyproject.toml for legal <change type> values.
 	 python -m towncrier.check
 
+         # put it back
+	 mv pyproject.toml towncrier.pyproject.toml
```

Comment:
Also, it's unrelated but it's so close by ... Perhaps it's worth taking this opportunity to add:

    whitelist_externals = 
        /bin/mv

and silence the "WARNING: test command found but not installed in testenv" message

If you'd rather not then I'll do a PR as soon as this one lands (my fault since I introduced this stuff).